parl/utils/logger.py

```diff
@@ -81,6 +81,13 @@ def _getlogger():
     logger = logging.getLogger('PARL')
     logger.propagate = False
     logger.setLevel(logging.DEBUG)
+
+    if 'DEBUG' in os.environ:
```

Comment:
Have you checked that all jobs have an this environmental variable and print log at the specified path?

Reply:
Yes, the job will output logs to this dir.